acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob

```diff
@@ -24,7 +24,7 @@ before_script:
     function makepot {
         # if you use weblate, remove --msgmerge here and install the msgmerge addon on weblate
         venv/bin/click-odoo-makepot -c odoo-ci.cfg -d ${DB_NAME} --commit --msgmerge --addons-dir=$(pwd)/odoo/addons
-        git push "git@gitlab.acsone.eu:${CI_PROJECT_PATH}" "HEAD:${CI_COMMIT_REF_NAME}"
+        git push -o ci.skip "git@gitlab.acsone.eu:${CI_PROJECT_PATH}" "HEAD:${CI_COMMIT_REF_NAME}"
```

Comment:
TIL `--push-option`

I kind of remember we did not do it because there was a potential race condition, but I can't tell exactly which one right now. @acsonefho  Have you tried it on some projects already?

Reply:
> TIL `--push-option`
> 
> I kind of remember we did not do it because there was a potential race condition, but I can't tell exactly which one right now. @acsonefho Have you tried it on some projects already?

@sbidoul Yes it's used on a project for few months and recently on 2 projects. Maybe we can wait few weeks to ensure we don't have any issue.